plugins/transport-grpc/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { VariableDictionary } from "@amplication/code-gen-types";
+import { join } from "path";
+
+export const envVariables: VariableDictionary = [
+  { GRPC_CLIENT_URL_PATH: "localhost:9090" },
```

Comment:
Is "localhost:9090" later replaced by something arriving from the plugin settings?

Reply:
@mulygottlieb No, this is the default value configured for the env variable.